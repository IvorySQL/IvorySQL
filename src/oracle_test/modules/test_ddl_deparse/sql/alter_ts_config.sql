--
-- ALTER TEXT SEARCH CONFIGURATION
--

CREATE TEXT SEARCH CONFIGURATION en (copy=english);

ALTER TEXT SEARCH CONFIGURATION en
  ALTER MAPPING FOR host, email, url, sfloat WITH simple;
