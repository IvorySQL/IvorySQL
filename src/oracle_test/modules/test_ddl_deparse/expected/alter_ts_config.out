--
-- ALTER TEXT SEARCH CONFIGURATION
--
CREATE TEXT SEARCH CONFIGURATION en (copy=english);
NOTICE:  DDL test: type simple, tag CREATE TEXT SEARCH CONFIGURATION
ALTER TEXT SEARCH CONFIGURATION en
  ALTER MAPPING FOR host, email, url, sfloat WITH simple;
NOTICE:  DDL test: type alter text search configuration, tag ALTER TEXT SEARCH CONFIGURATION
